spec/operators/mergeMapTo-spec.ts

```diff
@@ -264,42 +214,6 @@ describe('Observable.prototype.mergeMapTo', () => {
     expectSubscriptions(e1.subscriptions).toBe(e1subs);
   });
 
-  it('should mergeMapTo many cold Observable, with parameter concurrency=1', () => {
-    const values = {i: 'foo', j: 'bar', k: 'baz', l: 'qux'};
-    const e1 =     hot('-a-------b-------c---|                                        ');
-    const e1subs =     '^                                                            !';
-    const inner =  cold('----i---j---k---l---|                                        ', values);
-    const innersubs = [' ^                   !                                        ',
-                     '                     ^                   !                    ',
-                     '                                         ^                   !'];
-    const expected =   '-----i---j---k---l-------i---j---k---l-------i---j---k---l---|';
-
-    function resultSelector(oV: string, iV: string, oI: number, iI: number) { return iV; }
-    const result = e1.mergeMapTo(inner, resultSelector, 1);
-
-    expectObservable(result).toBe(expected, values);
-    expectSubscriptions(inner.subscriptions).toBe(innersubs);
-    expectSubscriptions(e1.subscriptions).toBe(e1subs);
-  });
-
-  it('should mergeMap to many cold Observable, with parameter concurrency=2', () => {
-    const values = {i: 'foo', j: 'bar', k: 'baz', l: 'qux'};
-    const e1 =     hot('-a-------b-------c---|                    ');
-    const e1subs =     '^                                        !';
-    const inner =  cold('----i---j---k---l---|                    ', values);
-    const innersubs = [' ^                   !                    ',
-                     '         ^                   !            ',
-                     '                     ^                   !'];
-    const expected =   '-----i---j---(ki)(lj)k---(li)j---k---l---|';
-
-    function resultSelector(oV: string, iV: string, oI: number, iI: number) { return iV; }
```

Comment:
ditto

Reply:
Oh... yeah, there's the same test below it "without resultSelector"